README.md

```diff
@@ -0,0 +1,19 @@
+# Accelerometer Instrument
+
+An app that plays music via the accelerometer input from a phone or tablet (or even an old MacBook!). To run, start up a local server 
```

Comment:
Lets advertise the github webpage: https://hellochar.github.io/phone-theremin/ !